publications/recommender.tex

```diff
@@ -52,21 +55,46 @@
 \end{keywords}
 
 \section{Introduction}
```

Comment:
there is no abstract in this journal?

Reply:
All the papers have an abstract. Also note that according to http://jmlr.org/mloss/mloss-info.html it has to be 4 pages long at most.